src/Component.js

```diff
@@ -0,0 +1,62 @@
+
+var __extends = (this && this.__extends) || function (d, b) {
```

Comment:
Any particular reason for adding this implementation of extends rather than using either [`utils.extend` from reflux-core](https://github.com/reflux/reflux-core/blob/master/src/utils.js#L19-L38) or `Object.assign`?